pkg/driver/azure-file/azure_file.go

```diff
@@ -65,9 +65,12 @@ func GetAzureFileGeneratorConfig() *generator.CSIDriverGeneratorConfig {
 			SidecarLocalMetricsPortStart:   commongenerator.AzureFileLoopbackMetricsPortStart + 1,
 			SidecarExposedMetricsPortStart: commongenerator.AzureFileExposedMetricsPortStart + 1,
 			Sidecars: []generator.SidecarConfig{
-				commongenerator.DefaultProvisioner.WithExtraArguments(
+				commongenerator.DefaultProvisionerWithSnapshots.WithExtraArguments(
 					"--extra-create-metadata=true",
 					"--timeout=300s",
+					"--worker-threads=100",
```

Comment:
The upstream Deployment does not set `--worker-threads`

Reply:
Removed.